CODEOWNERS

```diff
@@ -68,3 +68,6 @@
 # docs
 /man/openmp-utils.Rd @Anirban166
 /Seal_of_Approval.md @tdhock
+
+# GLCI
+.gitlab-ci.yml @jangorecki @ben-schwen
```

Comment:
I think this part is unrelated to the issue that this PR is meant to resolve. Also I think any changes to codeowners shouls be discussed before putting other people into it. I would like to assign myself to ci file but knowing I won't have time for really maintaining it in near future, I would not do so.

Reply:
@jangorecki I agree, I probably should have asked before putting your name there. But I don't see it so much as a burden that you have to review everything, but more as documentation (other than git blame) and a chance for us to get your thoughts and feedback (when needed) on changes there.

Reply:
Ah, sorry about that, I meant to require Jan's review before submitting here given the CODEOWNERS change.

And right, it's not a requirement for review, just that you're a better default assignee than me. What I had in mind is (1) PRs from Ben go to Jan for review (2) other PRs go to Ben for review.

Reply:
(FWIW, I was wondering if the above would be better served by doing `@ben-schwen @jangorecki` instead of `@jangorecki @ben-schwen`, but AFAICT there's no difference as both will be requested: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file)